test/TokenVesting.js

```diff
@@ -163,7 +170,7 @@ describe("TokenVesting", function () {
       // check that anyone cannot revoke a vesting
       await expect(
         tokenVesting.connect(addr2).revoke(vestingScheduleId)
-      ).to.be.revertedWith(" Ownable: caller is not the owner");
+      ).to.be.revertedWith("VM Exception while processing transaction: reverted with reason string 'Ownable: caller is not the owner'");
```

Comment:
Oh..I am not sure about this - the tests failed for me with error that they were expecting your version of the revert message, but got the fulle `VM Exception....` - this might be cause by my setup, but I am not sure. Thoughts?

Reply:
@abdelhamidbakhta Any thoughts on this?